index.js

```diff
@@ -67,6 +77,63 @@ function toggleColor(element) {
     }
 }
 
+header.addEventListener('click', event => {
+    console.log(event.target)
+    toggleColor(event.target)
+})
+
 /***** Deliverable 2 *****/
 
+likeButton = document.querySelector("button.like-button")
+likeCount = document.querySelector("p.likes")
+
+count = 1000
+likeButton.addEventListener('click', event => {
+    // My appraoch
+    // likeButton.classList.toggle('liked')
+    // count += 1;
+    // likeCount.innerHTML = "Likes: " + count
+    // APPROACH 1
+    const likesPDisplay = document.querySelector('p.likes')
+    traveler.likes += 1
+    likesPDisplay.textCOntent = `${traveler.likes} Likes`
+    console.log('clicked', event.target)
+
+    APPROACH 2
+    const likesPDisplay = document.querySelector('p.likes')
+    const currLikes = parseInt(likesPDisplay.textContent)
+    likesPDisplay.textContent = `${currLikes + 1} Likes`
+})
+
+// Approach 3
+console.log(event.target)
+const likesPDisplay = event.target.previousElementSibling
+console.log(likesPDisplay)
 /***** Deliverable 3 *****/
+
+const createAnimalSightingForm = document.querySelector('#new-animal-sighting-form')
+createAnimalSightingForm.addEventListener('submit', event => {
+    event.preventDefault()
+    console.log('form submitted')
+})
+
+
+const speciesInput = event.target.species.value
```

Comment:
This all looks great except it's not inside your event listener! I would toss lines 121 - 139 into its own function and call it inside your event listener.